src/http/endpoints/confirm.rs

```diff
@@ -0,0 +1,112 @@
+use crate::ayb_db::db_interfaces::AybDb;
```

Comment:
I like that we've pushed each endpoint into its own file. I haven't confirmed line-by-line, but can you confirm that there are no implementation changes in these files, just a cut/paste from `endpoints.rs`?

Reply:
I did Ctrl+X and Ctrl+V haha
I'll check it later, but it should be exactly the same